src/git.rs

```diff
@@ -129,31 +131,32 @@ pub struct Repository {
 
     ref_map: RefMap,
     head: Head,
+    commit_hashes: Vec<CommitHash>,
```

Comment:
Could you please comment that you're keeping it to preserve the original order of commits?

Reply:
Done in https://github.com/lusingander/serie/pull/5/commits/3c5bf73c7b2ee5e44a3517095c74d1d74c02996b